__tests__/mocks/doctor.ts

```diff
@@ -0,0 +1,58 @@
+export const mockDoctor = {
+    fullName: "John Doe",
+    about: "Experienced doctor",
+    gender: "Male",
+    email: "john.doe@example.com",
+    photoUrl: "https://samplehospital.com/doctors/john-doe.jpg",
+    yearsOfExperience: 10,
+    mainInstitutionId: "main-institution-id",
+    mainInstitutionName: "Sample Hospital",
+    specialities: ["Cardiology", "Neurology"],
```

Comment:
There is already a mock doctor data on the main. Since it is redundant to have to mock doctor data twice you should rebase from the main and use the mock data there is. Also update your test based on the mock data available

Reply:
its is not the same